embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java

```diff
@@ -80,6 +82,10 @@
         @ConfigDefault("false")
         public boolean getSkipGlacierObjects();
 
+        @Config("direct_path_prefix_object")
```

Comment:
It's subjective but this sounds ambiguous to me. What do you think about instead of this, we'll have a string config "single_object_path", then we just use it as the only path instead of having to combine 2 configs?

Reply:
`single_object_path` sounds more 'direct'  than above boolean configuration, but I'm afraid putting one more path fetching solution will make the branch and configuration become more complex to plugin users.

Reply:
I'm not totally aligned but this seems fine to me.